pkg/services/replicator/process.go

```diff
@@ -45,6 +49,11 @@ func (p *Replicator) HandleTask(ctx context.Context, task Task, res TaskResult)
 			stream = client.DemuxReplicatedObject(stream)
 		}
 	} else {
+		if err = p.checkExpiration(task.obj); err != nil {
+			p.log.Debug("expiration check", zap.Error(err))
```

Comment:
i'd `Info` (at least) cases with broken exp attributes cuz we don't expect them in general

Reply:
ok, added details and made it warn